presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestFeaturesConfig.java

```diff
@@ -91,7 +91,8 @@ public void testDefaults()
                 .setFilterAndProjectMinOutputPageRowCount(256)
                 .setUseMarkDistinct(true)
                 .setHistogramGroupImplementation(HistogramGroupImplementation.NEW)
-                .setArrayAggGroupImplementation(ArrayAggGroupImplementation.NEW));
+                .setArrayAggGroupImplementation(ArrayAggGroupImplementation.NEW)
+                .setDynamicFilteringEnabled(false));
```

Comment:
features flags should probably go with execution PR

Reply:
We should have them to keep optimizer disabled until execution is implemented.